jbmc/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ add_subdirectory(src)
 add_subdirectory(unit)
 
 add_custom_target(java-models-library ALL
-    COMMAND mvn package
+    COMMAND mvn --quiet -Dmaven.test.skip=true package
```

Comment:
:question: I'm with Chris - what is the reason for `Dmaven.test.skip=true` Doubt we have any tests in the core models and probably here isn't the place to run them if we do, but perhaps put this change in a separate commit so this can at least be justified.

Reply:
Many thanks, this should make it a lot clearer (and also easier to revert in future): done, please see whether the commit message is helpful.